src/electron/views/renderer-initializer.ts

```diff
@@ -339,10 +349,36 @@ getPersistedData(indexedDBInstance, indexedDBDataKeysToFetch).then(
 
         const fixInstructionProcessor = new FixInstructionProcessor();
 
+        const issueFilingController = new IssueFilingControllerImpl(
+            shell.openExternal,
```

Comment:
This will open the default browser, right?

Reply:
Yep!